### DISCUSSION TITLE
Keep RAF (x2 kill xp) lvl cap and server experience (x2 quests, exploration, RDF, & whatever BGs rates are) lvl cap consistent

### DISCUSSION THREAD

**Discussion by andovcr** [chromiecraft/chromiecraft#6209] [Suggestions]
Whenever the level cap for RAF is raised (in future brackets), could we please raise the level cap for x2 quest, exploration, & RDF to match it to keep the levels consistent with each other?  Currently the RAF cap is lvl 68, and the server's cap for quests is lvl 65.  I understand that these are two different systems, but that is not obvious to new players, nor is there a good reason that they should be intermittently different, given that they have been the same in past brackets.

If this is deemed not appropriate by the staff, could we please be provided an explanation for why?  Otherwise it comes off as introducing confusion to new players for no discernable reason.  I think transparency around decision-making and then communicating that in a consistent manner to the player-base will only aid the server's growth.

Additionally, could the website please reflect if there are any level caps associated with the increased BG rates?
<img width="769" alt="image" src="https://github.com/chromiecraft/chromiecraft/assets/58115742/23460f74-608a-47ad-a9fe-5a4181e9bb4c">

**Comment by Amrx2** [comment]
Hi @andovcr, this was an oversight on our behalf. This has been changed now so both RAF as well as the boosted quest XP rates run to 68 now. Thanks for flagging this.

**Reply by ProhorovIlja** [reply]
Will you be increasing it as level cap will be increasing up to level 80?

**Reply by shaszun** [reply]
That's like 2years into the future.
